prov/gni/test/rdm_sr.c

```diff
@@ -1234,12 +1251,13 @@ void do_send_buf(void *s, void *t, int len)
 	} while (!(source_done && dest_done));
 
 	rdm_sr_check_cqe(&s_cqe, t, (FI_MSG|FI_SEND), 0, 0, 0);
-	rdm_sr_check_cqe(&d_cqe, s, (FI_MSG|FI_RECV), t, len, 0);
-	rdm_sr_check_cntrs(1, 1, 0, 0);
+	rdm_sr_check_cqe(&d_cqe, p, (FI_MSG|FI_RECV), t, len, 0);
+	s[0] = 1; r[1] = 1;
+        rdm_sr_check_cntrs(s, r, s_e, r_e);
```

Comment:
A lot of the rdm_sr_check_cntrs() lines in this file replaced a tab indent with spaces.